urbanaccess/gtfs/load.py

```diff
@@ -227,6 +227,9 @@ def gtfsfeed_to_df(gtfsfeed_path=None,validation=False,verbose=True,bbox=None,re
                                                        trips_df=trips_df[['trip_id','route_id']],
                                                        info_to_append='route_type_to_stop_times')
 
+        # TODO: Appending seems improper; given the initial dataframes are empty we should either override or 
```

Comment:
The initial empty dataframes are created in order to use the append functions here. Append is nice in that dfs with different columns do not throw errors. The dfs being appended are all different dfs in a loop from different gtfs sources with structures that may vary. If you have suggestions for another method to go about this please let us know, we would be happy to test it out.